src/ZendCodingStandard/ruleset.xml

```diff
@@ -201,232 +70,823 @@
+    <rule ref="SlevomatCodingStandard.Commenting.ForbiddenAnnotations">
         <properties>
-            <property name="ignoreNewlines" value="true"/>
+            <property name="forbiddenAnnotations" type="array">
+                <element value="@api"/>
+                <element value="@author"/>
+                <element value="@category"/>
+                <element value="@created"/>
+                <element value="@package"/>
+                <element value="@subpackage"/>
+                <element value="@version"/>
+                <element value="@expectedException"/>
+                <element value="@expectedExceptionCode"/>
+                <element value="@expectedExceptionMessage"/>
+                <element value="@expectedExceptionMessageRegExp"/>
+            </property>
         </properties>
     </rule>
-    <!-- Forbid spaces before semicolon `;` -->
-    <rule ref="Squiz.WhiteSpace.SemicolonSpacing"/>
-    <!-- Forbid superfluous whitespaces -->
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace">
+
+    <!-- The words _private_, _protected_, _static_, _constructor_, _deconstructor_,
+         _Created by_, _getter_ and _setter_, MAY NOT be used in comments. -->
+    <rule ref="SlevomatCodingStandard.Commenting.ForbiddenComments">
         <properties>
-            <!-- turned on by PSR2 -> turning back off -->
-            <property name="ignoreBlankLines" value="false"/>
+            <property name="forbiddenCommentPatterns" type="array">
+                <element value="~^(?:(?!private|protected|static)\S+ )?(?:con|de)structor\.\z~i"/>
+                <element value="~^Created by .+\.\z~i"/>
+                <element value="~^(User|Date|Time): \S+\z~i"/>
+                <element value="~^\S+ [gs]etter\.\z~i"/>
+                <element value="~^Class \S+\z~i"/>
+            </property>
         </properties>
     </rule>
-    <rule ref="Squiz.WhiteSpace.SuperfluousWhitespace.EmptyLines">
-        <!-- turned off by PSR2 -> turning back on -->
-        <severity>5</severity>
+
+    <!-- The `@var` tag MAY be used in inline comments to document the _Type_
+         of properties. Single-line property comments with a `@var` tag SHOULD
+         be written as one-liners. The `@var` MAY NOT be used for constants. -->
+    <rule ref="SlevomatCodingStandard.Commenting.InlineDocCommentDeclaration"/>
+    <rule ref="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment"/>
+
+    <!-- The correct tag case of PHPDocs and PHPUnit tags MUST be used. -->
+    <rule ref="WebimpressCodingStandard.Commenting.TagCase"/>
+
+    <!-- Inline DocComments MAY be used at the end of the line, with at least a
+         single space. Inline DocComments MAY NOT be placed after curly brackets. -->
+    <rule ref="WebimpressCodingStandard.Commenting.Placement"/>
+    <rule ref="WebimpressCodingStandard.Commenting.NoInlineCommentAfterCurlyClose"/>
+
+    <!-- Typed parameters and return types SHOULD be avoided. -->
```

Comment:
Wait, what? I thought we _wanted_ type hints? 

Or is this to indicate we do not want _docblock comments_ for them if they already have explicit typehints?

Reply:
It's for the docblock comments :P

Reply:
Okay - so update the comment to indicate that. :)

And thanks!

Reply:
This text was a repetition for the same rule mentioned before. But since it required extra rules I have repeated the rule to group it with the sniffs. It was pulled out of context because of that. I've fixed the wording.